java/runtime/src/main/java/org/ray/runtime/DefaultRayRuntimeFactory.java

```diff
@@ -16,12 +12,11 @@
  */
 public class DefaultRayRuntimeFactory implements RayRuntimeFactory {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(DefaultRayRuntimeFactory.class);
+  private final Logger logger = LoggerFactory.getLogger(DefaultRayRuntimeFactory.class);
```

Comment:
loggers are better defined as static, otherwise they won't be accessible from static methods.

Reply:
e... if you need to use logger in a static method, then define the logger as static variable. static variable can be used in non-static method.